src/egon/data/datasets/fix_ehv_subnetworks.py

```diff
@@ -230,57 +225,87 @@ def fix_subnetworks(scn_name):
             6,
         )
 
-        # Missing 380kV line near Elsfleth
+        # Missing line from Umspannwerk Plottling to Gänsdorf UW
         add_line(
-            # Line
-            8.419326700000001,
-            53.229867000000006,
-            # Schaltanlage Elsfleth/West
-            8.402976949446648,
-            53.2371468322213,
+            12,
+            853318030996972,  # Umspannwerk Plottling
+            48,
+            76508569478315,
+            12,
+            769768646403532,  # Gänsdorf UW
+            48,
+            80533685376445,
             380,
             scn_name,
-            6,
+            3,
         )
 
-        # Missing 380kV line near Magdala
+        # Missing line inside Ottstedt
         add_line(
-            # Line north south
-            11.4298432,
-            50.9117467,
-            # Line east
-            11.4295305,
-            50.9115176,
+            11,
+            4295305,  # Ottstedt
+            50,
+            9115176,
+            11,
+            4299277,  # Ottstedt
+            50,
+            9114496,
```

Comment:
See comment above

Reply:
There was an inconsistency with "." and ",". Corrected in a4dd369747b662e5d2eb5b4ff3bc84f6097a0f4d